src/info/title.rs

```diff
@@ -81,3 +81,72 @@ impl std::fmt::Display for Title {
         }
     }
 }
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use git_repository::{open, Repository, ThreadSafeRepository};
+    use git_testtools;
+    use owo_colors::AnsiColors;
+
+    pub type Result<T = ()> = std::result::Result<T, Box<dyn std::error::Error>>;
```

Comment:
```suggestion
    type Result<T = ()> = std::result::Result<T, Box<dyn std::error::Error>>;
```
I don't think this needs to be `pub`. It's not going to be used outside of the `tests` mod.